python/golem_task_api/envs/__init__.py

```diff
@@ -0,0 +1,28 @@
+from google.protobuf.json_format import MessageToJson
+
+from golem_task_api.envs.constants import *
```

Comment:
i personally do not like `import *` is there something in our styleguide on this?

Would propose to `import constants`

Reply:
That's intentional. I would like re-export all env-related constants. But I can enumerate them as well.